doc/page01_info.md

```diff
@@ -185,19 +191,37 @@ $ ./output
 
 That's it, **EVE** is properly installed and ready to use.
 
+## Use in CMake
+
+Once installed, **EVE** may be consumed though its config-file **CMake** package. Simply find and
+link against **EVE's** **CMake** target, as you would any other **CMake** library, and then
+configure and build your **CMake** project.
+
+```cmake
+add_executable(use-eve main.cpp)
+find_package(eve CONFIG REQUIRED)
+target_link_libraries(use-eve PRIVATE eve::eve)
+```
+
+> If a custom installation prefix was used, ensure your **EVE** installation is within **CMake's**
+  search path with the use of the **CMake** variables **eve_ROOT** (CMake 3.12), **eve_DIR**, or
```

Comment:
Should we remove the 3.12 remark ?

Reply:
Since we are now saying 3.19 is the minimum, yes. Thanks!